Meshtastic/Views/Nodes/MeshMap.swift

```diff
@@ -92,6 +107,7 @@ struct MeshMap: View {
 									Logger.services.error("Unable to convert local point to coordinate on map.")
 									return
 								}
+								centerMapAt(coordinate: coordinate)
```

Comment:
This was originally a quick hack for testing (since I don't have a way to access node data from Simulator), but centering the map on the location a waypoint is being proposed actually felt very intuitive so I decided to open this PR.

https://github.com/user-attachments/assets/c6800669-1976-4a6d-980d-52686b53269e